### PR TITLE
As a potential customer I can see a list of services that they are offering, so that I know if they provide services which I need.

### PR DESCRIPTION
**Pull request by lspk129**
https://tree.taiga.io/project/troublemaker-website/us/2?milestone=133526

### CODE REVIEW

File: components/Service/ServiceContainer/index.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import ServiceContainer from './index';
+
+describe('ServiceLayout', () => {
```
Comment (tmxyz) [inline components/Service/ServiceContainer/index.test.js]: This is a ServiceContainer component, the name is wrong.

File: components/Service/ServiceHeader/index.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import ServiceHeader from './';
+
+describe('ServicesSection', () => {
```
Comment (tmxyz) [inline components/Service/ServiceHeader/index.test.js]: Wrong naming.

File: components/Service/ServiceItem/index.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import ServiceItem from './index';
+
+describe('ServiceList', () => {
```
Comment (tmxyz) [inline components/Service/ServiceItem/index.test.js]: Wrong naming.

File: components/Service/index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import ServiceHeader from './ServiceHeader';
+import ServiceContainer from './ServiceContainer';
+import ServiceItem from './ServiceItem';
+
+const ServiceSection = () => (
```
Comment (tmxyz) [inline components/Service/index.js]: Wrong name.

File: components/Service/index.test.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+describe('ServiceSection', () => {
+  it('renders correctly', () => {
+    const wrapper = renderer.create(<ServiceSection />);
+    const tree = wrapper.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders ServiceHeader component', () => {
+    const wrapper = shallow(<ServiceSection />);
+    expect(wrapper).toBePresent();
```
Comment (tmxyz) [inline components/Service/index.test.js]: You only need one of these.

File: components/Service/index.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  <div>
+    <ServiceHeader>
+      What we do
+    </ServiceHeader>
+    <ServiceContainer>
+      <ServiceItem>
+        <div className="web-dev">
```
Comment (tmxyz) [inline components/Service/index.js]: Rethink this.

File: components/Service/index.test.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    expect(wrapper.find('.web-dev').find('p')).toHaveText(
+      'We help you build fast, functional, performance oriented websites and web apps',
+    );
+  });
+
+  it('renders image of service', () => {
+    const wrapper = shallow(<ServiceSection />);
+    expect(wrapper).toBePresent();
+    expect(wrapper.find('.web-design').find('i')).toHaveClassName('material-icons');
```
Comment (tmxyz) [inline components/Service/index.test.js]: Also this should be done better.

File: components/Service/ServiceContainer/index.test.js (hunk 1 above)
Comment (lspk129) [inline components/Service/ServiceContainer/index.test.js]: Fixed.

File: components/Service/ServiceHeader/index.test.js (hunk 2 above)
Comment (lspk129) [inline components/Service/ServiceHeader/index.test.js]: Fixed.

File: components/Service/ServiceItem/index.test.js (hunk 3 above)
Comment (lspk129) [inline components/Service/ServiceItem/index.test.js]: Fixed.

File: components/Service/index.js (hunk 4 above)
Comment (lspk129) [inline components/Service/index.js]: Fixed.

File: components/Service/index.test.js (hunk 5 above)
Comment (lspk129) [inline components/Service/index.test.js]: Fixed.

File: components/Service/index.js (hunk 6 above)
Comment (lspk129) [inline components/Service/index.js]: Fixed.

File: components/Service/index.test.js (hunk 7 above)
Comment (lspk129) [inline components/Service/index.test.js]: Fixed.

**Pull request merged**